CHANGELOG.rst

```diff
@@ -12,9 +12,11 @@ All notable changes to this project are documented in this file.
 - Fixed missing uri locations in ``neo/api/REST/RestApi.py`` (`PR #342 <https://github.com/CityOfZion/neo-python/pull/342>`_)
 - Fixed privatenet check by fixing the chain path for checks in Settings (`PR #341 <https://github.com/CityOfZion/neo-python/pull/341>`_)
 - Fixed ``neo-privnet.sample.wallet``
+- Fix for current block height lag behind other RPC implementations by 1-3 blocks
 - Fixed ``bootstrap.py`` to use the specified data directory, instead of hard-coded relative paths.
 
 
+>>>>>>> 6f4b5d4feb45aa1d11ebde380e761e213b17765f
```

Comment:
@localhuman you probably didn't mean to include this merge conflict line 😀

Reply:
Good catch :)